src/Microsoft.Extensions.ServiceDiscovery.Abstractions/ServiceEndpoint.cs

```diff
@@ -11,35 +11,35 @@ namespace Microsoft.Extensions.ServiceDiscovery;
 /// <summary>
 /// Represents an endpoint for a service.
 /// </summary>
-[DebuggerDisplay("{GetEndPointString(),nq}")]
-public abstract class ServiceEndPoint
+[DebuggerDisplay("{GetEndpointString(),nq}")]
+public abstract class ServiceEndpoint
 {
     /// <summary>
     /// Gets the endpoint.
     /// </summary>
-    public abstract EndPoint EndPoint { get; }
+    public abstract EndPoint Endpoint { get; }
 
     /// <summary>
     /// Gets the collection of endpoint features.
     /// </summary>
     public abstract IFeatureCollection Features { get; }
 
     /// <summary>
-    /// Creates a new <see cref="ServiceEndPoint"/>.
+    /// Creates a new <see cref="ServiceEndpoint"/>.
     /// </summary>
-    /// <param name="endPoint">The endpoint being represented.</param>
+    /// <param name="endpoint">The endpoint being represented.</param>
     /// <param name="features">Features of the endpoint.</param>
-    /// <returns>A newly initialized <see cref="ServiceEndPoint"/>.</returns>
-    public static ServiceEndPoint Create(EndPoint endPoint, IFeatureCollection? features = null) => new ServiceEndPointImpl(endPoint, features);
+    /// <returns>A newly initialized <see cref="ServiceEndpoint"/>.</returns>
+    public static ServiceEndpoint Create(EndPoint endpoint, IFeatureCollection? features = null) => new ServiceEndpointImpl(endpoint, features);
 
     /// <summary>
-    /// Gets a string representation of the <see cref="EndPoint"/>.
+    /// Gets a string representation of the <see cref="Endpoint"/>.
     /// </summary>
-    /// <returns>A string representation of the <see cref="EndPoint"/>.</returns>
-    public virtual string GetEndPointString() => EndPoint switch
+    /// <returns>A string representation of the <see cref="Endpoint"/>.</returns>
+    public virtual string GetEndpointString() => Endpoint switch
     {
         DnsEndPoint dns => $"{dns.Host}:{dns.Port}",
         IPEndPoint ip => ip.ToString(),
```

Comment:
Nit. I know it's not the point of this PR, but it seems like the last two switch cases are the same.

```suggestion
```